bears/ruby/RuboCopBear.py

```diff
@@ -9,6 +9,22 @@
 from coalib.results.Result import Result
 
 
+def map_naming_convention_to_rubocop(setting):
+    '''
+    This function converts the string ``setting`` (that refers to a naming
+    convention) into a ``rubocop`` specific naming convention.
+
+     >>> map_naming_convention_to_rubocop('snake')
+     'snake_case'
+     >>> map_naming_convention_to_rubocop('camel')
+     'camelCase'
+     '''
+    if setting == 'camel':
+        return 'camelCase'
+    else:
+        return 'snake_case'
```

Comment:
nah you should check for more and error out properly if it's neither camel or snake IMO